tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -692,6 +693,10 @@ EOF
 		popd
 	fi
 
+	if [ "${KATA_BUILD_CC}" == "yes" ]; then
+		install -o root -g root -m 0755 init_trusted_storage.sh "${ROOTFS_DIR}/usr/local/bin/"
```

Comment:
- Random thought: what about a context (`--context=`) if the image supports SELinux?
- Can't the perms be `0750` (or even `0500`)?